cmd/minikube/cmd/root.go

```diff
@@ -80,9 +80,13 @@ func init() {
 
 // initConfig reads in config file and ENV variables if set.
 func initConfig() {
+	configPath := constants.MakeMiniPath("config")
 	viper.SetConfigName("config")
-	viper.AddConfigPath(constants.MakeMiniPath("config"))
-	viper.ReadInConfig()
+	viper.AddConfigPath(configPath)
```

Comment:
unfortunately viper doesn't expose the path its reading from, just the config file its read (which is unhelpful if it can't find one)

https://godoc.org/github.com/spf13/viper#ConfigFileUsed